src/PhpSpreadsheet/Calculation/LookupRef.php

```diff
@@ -647,7 +647,7 @@ public static function INDEX($arrayValues, $rowNum = 0, $columnNum = 0)
                     if (isset($arrayColumn[$rowNum])) {
                         $returnArray[] = $arrayColumn[$rowNum];
                     } else {
-                        return $arrayValues[$rowNum];
+                        return array( $rowNum => $arrayValues[$rowNum] );
```

Comment:
array should write to []

Reply:
array() in this case is much intuitive. But if you insist... :)

Reply:
cause phpspreadsheet requires php5.6 and newer, the new feature recommend use [] instead of array(), or it won't pass the CI system.